source/install-atlas-cli.txt

```diff
@@ -143,32 +143,6 @@ Follow These Steps
 
                   .. tabs::
 
-                     .. tab:: RHEL
```

Comment:
other versions of rhel are still supported, tho I'm not sure these are the same instructions